components/gly_gen/glycosight/data/index.ts

```diff
@@ -0,0 +1,21 @@
+import path from 'path'
+import { uploadFile } from '@/components/core/file/api/upload'
+import type { SessionWithId } from '@/app/pages/api/auth/[...nextauth]'
+
+export const exampleFile = path.resolve(
+  process.env.APP_ROOT as string,
+  'components',
+  'gly_gen',
+  'glycosight',
+  'data',
+  '01CPTAC_OVprospective_G_JHUZ_20160317_QE_r01.mzid.gz',
+)
+
+export type UploadExampleFileResponse = Awaited<ReturnType<typeof uploadExampleFile>>
+
+export async function uploadExampleFile(session?: SessionWithId) {
+  return await uploadFile({
+    url: `${process.env.PUBLIC_URL}/api/v1/components/gly_gen/glycosight/`,
+    filename: '',
```

Comment:
Should probably give it a filename

Reply:
Seems reasonable, will double check if it has been carried through.